build.sh

```diff
@@ -71,6 +71,8 @@ PRINT_USAGE() {
     echo "                       Disable FEATUREs from JSRT experimental features."
     echo "     --valgrind        Enable Valgrind support"
     echo "                       !!! Disables Concurrent GC (lower performance)"
+    echo "     --use-ccache      Enable use of ccache."
+    echo "     --ccache=NAME     Specify the ccache binary name.  Default: ccache"
```

Comment:
is it possible to only add one flag here instead of two? If you were to just pass in `--ccache`, it could assume that the default program name will be `ccache`, or if you pass in `--ccache=X`, then it could take X as the ccache executable.

Reply:
Yes, I think that's a good idea.  I was struggling a bit with naming these - unifying them feels much better!

Reply:
Look at how `-j` is implemented to get an idea -- theres one case for `-j` and another for `-j=X`, if I remember correctly, which makes some of the bash messiness go away.